Geometry/HGCalGeometry/test/python/testHGCalWafer_cfg.py

```diff
@@ -0,0 +1,47 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("PROD")
+process.load("SimGeneral.HepPDTESSource.pdt_cfi")
+process.load("Configuration.Geometry.GeometryExtended2023D46Reco_cff")
```

Comment:
remember to fix this when D46 is officially added as 2026...

Reply:
Yes I shall remember